src/validation/userValidation.js

```diff
@@ -5,9 +5,9 @@ const userValidation = [
     .withMessage('First name field cannot be empty.'),
   check('lastName').trim().not().isEmpty()
     .withMessage('last name field cannot be empty.'),
-  check('userName').trim().not().isEmpty()
+  check('username').trim().not().isEmpty()
     .withMessage('username field cannot be empty.'),
-  check('userName').isAlphanumeric().withMessage('username should contain only letters and numbers'),
+  check('username').isAlphanumeric().withMessage('username should contain only letters and numbers'),
```

Comment:
Please return all these fields(userName) to what they were

Reply:
We have dropped username field entirely from the user object